ci/builder/requirements.txt

```diff
@@ -31,9 +31,10 @@ pdoc3==0.10.0
 pg8000==1.29.4
 prettytable==3.5.0
 psutil==5.9.4
-# psycopg2 intentionally omitted. Use pg8000 instead.
+psycopg2==2.9.6
```

Comment:
Hi @benesch,
Do you mind me adding this dependency here? Also see the comment in line 3.
It is used in the cloud repo and I did not experience any issues on my Mac M2.

CC @philip-stoev

Reply:
I personally would need to use `psycopg2` for benchmarking and other time and CPU-sensitive stuff, so it would be very good if we decide it is acceptable in the Mz repository as well.

Reply:
We also need https://pypi.org/project/psycopg-binary/ I think if we are going all-in ?

Reply:
@philip-stoev: Can we merge it like this and do the upgrade in a follow-up PR?